docs/tools/glaze/example.md

```diff
@@ -0,0 +1,271 @@
+
+    ```sh
+    glaze did:create
+    ```
+
+    You can then set the displayed key as the `DID_KEY` environment variable when running other commands, or make sure to provide it using the `--key` flag, as used in the following steps.
+
+## 3. DataModel creation
+
+### SimpleNote schema
+
+We'll first create a JSON schema for an object containing a `text` property.
+
+=== "Script"
+
+    ```js
+    const noteSchemaID = await manager.createSchema('SimpleNote', {
+      $schema: 'http://json-schema.org/draft-07/schema#',
+      title: 'SimpleNote',
+      type: 'object',
+      properties: {
+        text: {
+          type: 'string',
+          title: 'text',
+          maxLength: 4000,
+        },
+      },
+    })
+    ```
+
+=== "CLI"
+
+    Here we will use the `model:add` command, that takes the following 4 arguments:
+
+    1. The local model name
+    2. The type to add, here a `schema`
+    3. The alias for this schema, here `SimpleNote`
+    4. The schema contents, as a JSON-encoded string
+
+    ```sh
+    glaze model:add simple-note schema SimpleNote '{"$schema":"http://json-schema.org/draft-07/schema#","title":"SimpleNote","type":"object","properties":{"text":{"type":"string","title":"text","maxLength":4000}}}' --key=<your key>
+    ```
+
+Now that we have a schema, we can use it as reference in the following steps.
+
+### myNote definition
+
+Definitions need to reference a specific version of a schema using a commit ID in URL form (starting with `ceramic://`).
+
+=== "Script"
+
+    Using the manager, schema URLs should be accessed using the `getSchemaURL()` method:
+
+    ```js
+    // Create the definition using the created schema ID
+    await manager.createDefinition('myNote', {
+      name: 'My note',
+      description: 'A simple text note',
+      schema: manager.getSchemaURL(noteSchemaID),
+    })
+    ```
+
+=== "CLI"
+
+    Before we can create a definition, we need to inspect the model to retrieve the commit ID of our schema, identified by the `version`:
+
+    ```sh
+    glaze model:inspect simple-note
+    ```
+
+    We can then copy the `version` associated to the `SimpleNote` schema to the following command:
+
+    ```sh
+    glaze model:add simple-note definition myNote '{"name":"My note","description":"A simple text note","schema":"ceramic://<SimpleNote schema version ID>"}' --key=<your key>
+    ```
+
+### exampleNote tile
+
+Creating a tile is similar to a definition, except for the `schema` reference that needs to be provided as metadata rather than in the content:
+
+=== "Script"
+
+    ```js
+    // Create a tile using the created schema ID
+    await manager.createTile('exampleNote',
+      { text: 'A simple note' },
+      { schema: manager.getSchemaURL(noteSchemaID) },
+    )
+    ```
+
+=== "CLI"
+
+    ```sh
+    glaze model:add simple-note tile exampleNote '{"text":"A simple note"}' --schema='ceramic://<SimpleNote schema version ID>' --key=<your key>
```

Comment:
It might be nice to include how windows users need to do this, we've seen enough messages about it in Discord over the past months. Not important by any means but could be nice.